core/templates/core/home_page.html

```diff
@@ -21,7 +21,7 @@
                             {% if article.editors_pick %}
                                 <div class="editors-pick"><div>Editors Pick</div></div>
                             {% endif %}
-                            {% image article.main_image original as background_image %}
+                            {% image article.main_image fill-2048x1365 as background_image %}
```

Comment:
After testing it does keep the row height, but since it is using the larger image, it isn't recropped to fit nicely in the space.